packages/frontend/src/view/pages/user/components/ActionsTable.tsx

```diff
@@ -11,56 +17,61 @@ interface ActionsTableProps {
 }
 
 export interface WithdrawableAssetEntry {
-  readonly icon: string
-  readonly symbol: string
+  readonly asset: Asset
   readonly amount: bigint
 }
 
 export function ActionsTable(props: ActionsTableProps) {
   return (
     <div className="mb-12 flex w-full flex-col rounded-lg border border-solid border-brand bg-gray-800 p-6">
       <p className="text-sm font-semibold text-zinc-500">Withdrawable assets</p>
-      {props.withdrawableAssets.map((asset) => (
-        <div
-          className="mt-3 flex items-center justify-between"
-          key={asset.symbol}
-        >
-          <div className="flex">
-            <p>Icon</p>
-            <p className="text-base ml-3 font-bold text-white">
-              {asset.symbol}
+      {props.withdrawableAssets.map((asset) => {
+        const assetInfo = assetToInfo(asset.asset)
+        const symbol =
+          assetInfo.symbol.length > 7
+            ? `${assetInfo.symbol.substring(0, 7)}...`
+            : assetInfo.symbol
+        return (
+          <div
+            className="mt-4 flex items-center justify-between"
```

Comment:
add some spacing, e.g. `gap-2`

Reply:
I am not sure what you mean

Reply:
Currently if the text is long there is no visual space between it, the asset symbol and the button. Adding the `gap-2` class on the div will add this spacing.